ts/features/payments/onboarding/hooks/useWalletOnboardingWebView.tsx

```diff
@@ -49,6 +49,7 @@ export const useWalletOnboardingWebView = (
   props: WalletOnboardingWebViewProps
 ) => {
   const { onSuccess, onError, onFailure, onDismiss } = props;
+  const [isLoading, setIsLoading] = React.useState(false);
```

Comment:
```suggestion
  const [isWebviewLoading, setIsWebviewLoading] = React.useState(false);
```
I think that, since we have a couple of "loading" cases in the component(see line 114 for example), and this hook is used in combination with components with their own loading states, this would help readability up in some cases; what do you think?